x-pack/test/functional/services/ml/settings_calendar.ts

```diff
@@ -18,17 +18,26 @@ export function MachineLearningSettingsCalendarProvider(
   const retry = getService('retry');
   const comboBox = getService('comboBox');
 
+  const localeCompare = (a: string, b: string) => a.localeCompare(b);
+
   return {
-    async assertCalendarRowJobs(calendarId: string, expectedConnectedJobs: string[]) {
+    async assertCalendarRowJobs(
+      calendarId: string,
+      expectedConnectedJobs: string[]
+    ): Promise<void> {
       await this.filterWithSearchString(calendarId, 1); // makes sure the table only has the one row we want to check
       const calendarRow = (await this.parseCalendarTable())[0];
-      const actualJobs: string[] = calendarRow.jobs.replaceAll(',', '').split(' ');
+      const sortedActual: string[] = calendarRow.jobs
+        .replaceAll(',', '')
+        .split(' ')
+        .sort(localeCompare);
 
-      const expectedWithinActual = expectedConnectedJobs.every((expected) =>
-        actualJobs.some((actual) => actual === expected)
-      );
+      const sortedExpected = expectedConnectedJobs.sort(localeCompare);
 
-      expect(expectedWithinActual).to.be(true);
+      expect(sortedActual).to.eql(
+        sortedExpected,
+        `Expect [${sortedActual}] to eql [${sortedExpected}]`
```

Comment:
This extra message basically just repeats the default message and doesn't add any extra information. We should rather make it something like
```
Expected job column entries for calendar ${calendarId} to be ${sortedExpected}, got ${sortedActual}
```
In case of a failure scenario, this would give us way better pointers of where to start the investigation.

Reply:
I hope [this](https://github.com/elastic/kibana/pull/178553/commits/9b11596c5876671dc8a9732cc5e58b2ced011a99) satisfies boss!